integrations/symfony-bundle.rst

```diff
@@ -0,0 +1,176 @@
+Symfony Bundle
+==============
+
+The usage documentation is split into two parts. First we explain how to configure the bundle in an application. The second part is for developing reusable Symfony bundles that depend on an HTTP client defined by the Httplug interface.
+
+For information how to write applications with the services provided by this bundle, have a look at the [Httplug documentation](http://docs.php-http.org).
+
+
+Use in Applications
+-------------------
+
+Custom services
+```````````````
+
++----------------------------------+---------------------------------------------------------------------+
+| Service id                       | Description                                                         |
++==================================+=====================================================================+
+| httplug.message_factory          | Service* that provides the `Http\Message\MessageFactory`            |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.uri_factory              | Service* that provides the `Http\Message\UriFactory`                |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.stream_factory           | Service* that provides the `Http\Message\StreamFactory`             |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.client.[name]            | | This is your Httpclient that you have configured.                 |
+|                                  | | With the configuration below the name would be `acme_client`.     |
++----------------------------------+---------------------------------------------------------------------+
+| httplug.client                   | This is the first client configured or a client named `default`.    |
++----------------------------------+---------------------------------------------------------------------+
+| | httplug.plugin.content_length  | | These are plugins that are enabled by default.                    |
+| | httplug.plugin.decoder         | | These services are not public and may only be used when configure |
```

Comment:
These services are not public and can only be used to configure

(its not may, private services are just not available after the container is build. "when configure" is not correct, it would be "when configuring", but "to configure" is active and thus easier to understand)